models/DataObject/Service.php

```diff
@@ -1890,6 +1892,11 @@ public static function getCsvData($requestedLanguage, LocaleServiceInterface $lo
                 }
 
                 $rowData = self::getCsvDataForObject($object, $requestedLanguage, $fields, $helperDefinitions, $localeService, false, $context);
+                foreach ($rowData as &$fieldData) {
+                    if($translator !== null) {
+                        $fieldData = $translator->trans($fieldData, [], 'admin', $requestedLanguage);
+                    }
+                }
```

Comment:
```suggestion
                if($translator !== null) {
                    foreach ($rowData as &$fieldData) {
                        $fieldData = $translator->trans($fieldData, [], 'admin', $requestedLanguage);
                    }
                }
```

We don´t have to do the foreach if there is no translator, right?